src/elasticsearch/addl_index_transformations/portal/demo_partonomy.py

```diff
@@ -0,0 +1,60 @@
+import requests
+from pathlib import Path
+from json import loads
+
+
+def build_tree_index():
+    '''
+    Returns a tuple:
+        - A tree, where each node has value, parent_id, and children.
+        - A dict indexing into every node of the tree by id.
+    '''
+    partonomy_path = Path(__file__).parent / 'cache/partonomy.jsonld'
+    if not partonomy_path.exists():
+        partonomy_url = 'https://cdn.jsdelivr.net/gh/hubmapconsortium/hubmap-ontology@1.0.0/ccf-partonomy.jsonld'
```

Comment:
Switch URL to https://cdn.jsdelivr.net/gh/hubmapconsortium/hubmap-ontology@1/ccf-partonomy.jsonld so that it gets the latest version (a new version may appear at end of March but definitely end of May).

Reply:
@bherr2 : I don't think that's the behavior we want: I really prefer to pin dependencies and have the behavior of code be consistent until we make an explicit change.

If this doesn't seem correct to you, let's talk on the call this afternoon.

Reply:
Your call. Not how I'd do it, but I understand your reasoning.